# Request for setting a named target
string target_name
---
bool isreturnsuccess